src/ossf_scorecard/contrib/models.py

```diff
@@ -0,0 +1,160 @@
+import attr
```

Comment:
This should not be in `src/ossf_scorecard/contrib/models.py`, rather `src/ossf_scorecard/models.py`?